sqlmesh/dbt/adapter.py

```diff
@@ -355,3 +362,54 @@ def _normalize(self, input_table: exp.Table) -> exp.Table:
             normalized_table.set("db", normalized_table.this)
             normalized_table.set("this", None)
         return normalized_table
+
+
+class CachingAdapter:
```

Comment:
I'm a little confused: why do we have cache here but also in the RuntimeAdapter? Do we even use this cache right now or is this still WIP?

Reply:
Oh, I suppose this should be `CachingEngineAdapter`, not dbt adapter.

Reply:
Ah, I forgot to update this part it seems 🤦 

My intention was to do `self.engine_adapter = CachingEngineAdapter(...)` in the `RuntimeAdapter`'s init method.

Reply:
Updated in https://github.com/TobikoData/sqlmesh/pull/1863/commits/68fdc54372c59e7b27601112df0ed6c153a6d9b2, please take another look and let me know if something looks off.